src/display/process/allProcs.go

```diff
@@ -283,11 +295,46 @@ func AllProcVisuals(dataChannel chan []*proc.Process,
 						runAllProc = true
 						killSelected = false
 						updateProcs()
+						sendSignal = false
+						updateUI()
+					case "j", "<Down>":
+						signals.Table.ScrollDown()
+						ui.Render(signals)
+					case "k", "<Up>":
+						signals.Table.ScrollUp()
+						ui.Render(signals)
+					case "1", "2", "3", "4", "5", "6", "7", "8", "9":
```

Comment:
I noticed an issue with the numeric navigation. `11`, `19`, etc. all work but `10` does not. It stays at 1. Similarly for `20` and `30`.

Adding `"0"` to this case fixes that

Reply:
Done!